cmd/omegaup-grader/frontend_handler.go

```diff
@@ -64,6 +65,10 @@ func updateDatabase(
 	db *sql.DB,
 	run *grader.RunInfo,
 ) {
+	contestScore := base.RationalToFloat(run.Result.ContestScore)
+	if run.PartialScore && contestScore != 1 {
```

Comment:
argh, me equivoqué. esto era

```suggestion
	if !run.PartialScore && contestScore != 1 {
```

D:

Reply:
D: 

Lo hago en otro cambio? o se va a revertir este?

Reply:
hazlo en otro cambio porfa